ui/src/app/workflows/components/workflow-dag/workflow-dag.tsx

```diff
@@ -397,10 +394,10 @@ export class WorkflowDag extends React.Component<WorkflowDagProps, WorkflowDagRe
         const graph = new dagre.graphlib.Graph();
 
         graph.setGraph({
-            edgesep: 20 / this.scale,
-            nodesep: 50 / this.scale,
+            edgesep: this.gap,
+            nodesep: this.gap,
             rankdir: this.state.horizontal ? 'LR' : 'TB',
-            ranksep: 50 / this.scale
+            ranksep: this.gap
```

Comment:
Does this change how the graph looks? Could you add a screenshot?

Reply:
this makes it more compact - moving from 50px sep to 32px sep (default zoom)